lib/v8.js

```diff
@@ -80,3 +87,110 @@ exports.getHeapSpaceStatistics = function() {
 
   return heapSpaceStatistics;
 };
+
+/* V8 serialization API */
```

Comment:
@bnoordhuis I would say C-style comments feel a bit more heading-y than C++-style comments? I never consciously noticed but I think I use `//` for text that only refers to the next one or two statements, whereas `/* … */` refers to a longer section of code. That also seems to match how we use `eslint-disable` comments in our codebase.

If you feel strongly about it, I have no problem changing the format in either way. :)